models/topics.models.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection");
+
+const findTopics = () => {
+  return db.query(`SELECT * FROM topics;`);
+};
```

Comment:
You are returning the whole response here including lots of meta data that you really do not care about - you should only return what is on the key of rows. Use a .then here and just return rows. You can then amend your test to just check topics.